test_scripts/Resumption/InteriorVehicleData/020_InteriorVD_resumption_two_apps_multiple_moduleTypes_IGN_OFF.lua

```diff
@@ -0,0 +1,76 @@
+---------------------------------------------------------------------------------------------------
+-- Proposal: https://github.com/smartdevicelink/sdl_evolution/blob/master/proposals/0188-get-interior-data-resumption.md
+--
+-- Description: Successful resuming of interior vehicle data for two apps after IGN_OFF in case apps are subscribed to
+--  the same and different moduleTypes
+--
+-- Precondition:
+-- 1. HMI and SDL are started
+-- 2. Mobile app1 and app2 with REMOTE_CONTROL hmi type are registered and activated
+-- 3. App1 is subscribed to module_1 and module_2
+-- 4. App2 is subscribed to module_2 and module_3
+--
+-- Sequence:
+-- 1. IGN_OFF and IGN_ON are performed
+-- 2. Apps start registration with actual hashIds after SDL restart
+-- SDL does:
+-- - a. send RC.GetInteriorVehicleData(module_1, default_moduleId),
+--    RC.GetInteriorVehicleData(module_2, default_moduleId)
+--    and RC.GetInteriorVehicleData(module_3, default_moduleId) to HMI during resumption data
+-- - b. respond RAI(SUCCESS) to both mobile apps
```

Comment:
b. SDL get a successful response for every GetInteriorVehicleData request with isSubscribed=true from HMI
c. SDL respond RAI(success=true, result code = SUCCESS) to both mobile apps
d. update hashId after successful resumption

Reply:
Done